src/main/java/com/czertainly/api/interfaces/core/web/OidEntryController.java

```diff
@@ -0,0 +1,62 @@
+package com.czertainly.api.interfaces.core.web;
+
+import com.czertainly.api.exception.NotFoundException;
+import com.czertainly.api.interfaces.AuthProtectedController;
+import com.czertainly.api.model.client.certificate.SearchRequestDto;
+import com.czertainly.api.model.core.oid.OidEntryListResponseDto;
+import com.czertainly.api.model.core.oid.OidEntryResponseDto;
+import com.czertainly.api.model.core.oid.OidEntryRequestDto;
+import com.czertainly.api.model.core.oid.OidEntryUpdateRequestDto;
+import com.czertainly.api.model.core.search.SearchFieldDataByGroupDto;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("/v1/oids")
+@Tag(name = "OID Management", description = "OID Management API")
+public interface OidEntryController extends AuthProtectedController {
```

Comment:
Shouldn't be this names as `CustomOidEntryController`? This endpoint is used to managed only custom OIDs, system or standardized OIDs cannot be changed.